bitnami/kube-state-metrics/README.md

```diff
@@ -197,8 +199,11 @@ The command removes all the Kubernetes components associated with the chart and
 | `serviceMonitor.relabelings`                    | ServiceMonitor relabelings                                                                                                                                         | `[]`                         |
 | `serviceMonitor.metricRelabelings`              | ServiceMonitor metricRelabelings                                                                                                                                   | `[]`                         |
 | `serviceMonitor.labels`                         | Extra labels for the ServiceMonitor                                                                                                                                | `{}`                         |
+| `selfMonitor.enabled`                           | Creates a selfMonitor to monitor kube-state-metrics itself                                                                                                         | `false`                      |
+| `selfMonitor.telemetryPort`                     | Kube-state-metrics telemetry Port                                                                                                                                  | `8081`                       |
+| `selfMonitor.telemetryNodePort`                 | Kube-state-metrics Node Port                                                                                                                                       | `""`                         |
+
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example the following command sets the `replicas` of the kube-state-metrics Pods to `2`.
```

Comment:
There is a minor typo here. Could you check it?

Reply:
@dgomezleon 
I don't delete the sentence.
**bitnami-bot** delete the sentence automatically.

Reply:
After I added the sentence back, bitnami-bot automatically deleted it again

Reply:
You are right. We are working on a fix [here](https://github.com/bitnami-labs/readme-generator-for-helm/pull/61)

Reply:
Please add the line again. `readme-generator` shouldn't remove it again